composer.json

```diff
@@ -34,7 +34,7 @@
         "propel/propel1": "~1.7",
         "symfony/yaml": "2.*",
         "symfony/translation": "~2.0",
-        "symfony/validator": "~2.0",
+        "symfony/validator": "~2.0, <2.2",
```

Comment:
> i know. but there is an incompatibility with the 2.2 version 

With all 2.2.x versions or this was fixed on a new patch? Because in this case, you should better use the `conflict` section.

I agree with @Ener-Getick, this solution would not be accepted at all.

What is the related issue?